server/broadcast-message-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "start": "2021-11-02T15:00:00+00:00",
-  "end": "2021-11-02T20:00:00+00:00",
-  "message": "From 6:00pm until 7:00pm today, it will not be possible to make referrals or record activity details in Refer and monitor an intervention. This is due to nDelius maintenance work."
+  "start": "2021-11-18T12:00:00+00:00",
+  "end": "2021-11-21T12:00:00+00:00",
```

Comment:
Should we make this end on Monday midnight? `2021-11-22T00:00`?

Reply:
The ticket specifies Sunday morning: https://trello.com/c/gQ491Tof/151-add-banner-to-service-due-to-delius-downtime-20th-november

But you're right, Sunday midnight is better since it gives a bit more leeway.